app/components/Views/SendFlow/SendTo/index.js

```diff
@@ -358,8 +358,35 @@ class SendFlow extends PureComponent {
     });
     this.toggleFromAccountModal();
   };
+  /**
+   * This returns the address name from the address book or user accounts if the selectedAddress exist there
+   * @param {*} toSelectedAddress - Represents the value of what user writes on toAddress input
```

Comment:
Is `toSelectedAddress` a `string`? Add the correct data type.

```suggestion
   * @param {String} toSelectedAddress - Represents the value of what user writes on toAddress input
```